csharp/src/Microsoft.AI.MachineLearning/Microsoft.AI.MachineLearning.targets

```diff
@@ -6,26 +6,26 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <RuntimesDirectory Condition="'$(OutputType)' == 'AppContainerExe'">$(MSBuildThisFileDirectory)..\..\runtimes\win-$(WindowsAI-Platform)\uap</RuntimesDirectory>
-    <RuntimesDirectory Condition="'$(RuntimesDirectory)' == ''">$(MSBuildThisFileDirectory)..\..\runtimes\win-$(WindowsAI-Platform)\native</RuntimesDirectory>
+    <RuntimesDirectory Condition="'$(TargetPlatformIdentifier)' == 'UAP'">$(MSBuildThisFileDirectory)..\..\runtimes\win-$(WindowsAI-Platform)\lib\uap10.0</RuntimesDirectory>
+    <RuntimesDirectory Condition="'$(TargetPlatformIdentifier)' != 'UAP'">$(MSBuildThisFileDirectory)..\..\runtimes\win-$(WindowsAI-Platform)\_native</RuntimesDirectory>
     <WindowsAIBinarySubfolder Condition="'$(UseWindowsMLStaticRuntime)' == 'true'">static\</WindowsAIBinarySubfolder>
     <WindowsAIBinary>$(RuntimesDirectory)\$(WindowsAIBinarySubfolder)Microsoft.AI.MachineLearning.dll</WindowsAIBinary>
+    <WindowsAIWinMD>$(MSBuildThisFileDirectory)..\..\lib\uap10.0\Microsoft.AI.MachineLearning.winmd</WindowsAIWinMD>
+    <WindowsAIExperimentalWinMD>$(MSBuildThisFileDirectory)..\..\lib\uap10.0\Microsoft.AI.MachineLearning.Experimental.winmd</WindowsAIExperimentalWinMD>
```

Comment:
This is needed?

Reply:
Yeah it needs to be copied to the output directory